content/sensu-go/6.3/operations/manage-business-services/_index.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Manage Business Services"
+description: "Sensu's business service management allows you to configure service components and rule templates for monitoring your business services."
+product: "Sensu Go"
+version: "6.3"
+weight: 55
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.3:
+    parent: operations
+    identifier: manage-business-services
+---
+
+Sensu's business service management allows you to monitor every component in your system with a top-down approach that produces meaningful alerts, prevents alert fatigue, and helps you focus on your core business services.
+Use components to identify elements of your business services and define monitoring rules that produce events for those components based on customized evaluation expressions.
+
+For example, with business service management, you can specify that failure of certain individual components will trigger an alert to an operator mailbox and create a ticket to address the next workday.
+At the same time, for failure of a component that affects availability or performance of your entire business service, your rule template can trigger an alert to your on-call staff for immediate response.
+You can also set a delay so that a component failure causes alerts only after the failure persists for a certain amount of time or after a number of polling intervals.
+
+Business service management highlights which business services are affected by the failure of an underlying system.
+
+Business service management requires two resources that work together to achieve top-down monitoring: components and rule templates.
+
+## Components
+
+Business service [components][1] are resources for defining and managing elements of a business service.
+Here is an example component definition that references the `status-threshold` rule template:
+
```

Comment:
Since this is the same example I used in the components reference, maybe we should use a different example here.